solr/solrj/src/resources/DeprecatedSystemPropertyMappings.properties

```diff
@@ -28,3 +28,5 @@ basicauth=solr.security.auth.basicauth.credentials
 cloud.solr.client.max.stale.retries=solr.solrj.cloud.max.stale.retries
 configset.upload.enabled=solr.configset.upload.enabled
 solr.hidden.sys.props=solr.responses.hidden.sys.props
+bootstrap_confdir=solr.configset.bootstrap.confdir
+collection.config.name=solr.collection.config.name
```

Comment:
The actual system property that was used is "collection.configName" not "collection.config.name".